exporter/googlecloudexporter/resource_mapper_test.go

```diff
@@ -141,7 +141,7 @@ func TestResourceMapper(t *testing.T) {
 			},
 			// Resource without matching config should be converted via default implementation
 			wantResource: &monitoredres.MonitoredResource{
-				Type: "global",
+				Type: "generic_task",
```

Comment:
Note that this behavior was a bug, and was fixed by https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/pull/286